### PR TITLE
🗺 enhance: Store location map integration on seller setup wizard

### LINKED ISSUE #1270: google maps is missing
google maps is missing on vendor setup

### PR DESCRIPTION
**Pull request by devAsadNur**
### All Submissions:

* [x] My code follows the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [ ] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Changes proposed in this Pull Request:
Previously, there were no store location map selection fields on the store setup wizard. Now, the fields are added by this PR.

* Closes #1270.

### How to test the changes in this Pull Request:

1. Go to the seller setup wizard.
2. Store location map fields should be there.

### Changelog entry

> enhance: Store location map integration on the seller setup wizard
>> Introducing a new enhancement in the seller setup wizard: seamless integration of a store location map. This enhancement allows sellers to effortlessly navigate and locate their store's position within the wizard interface.

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

### CODE REVIEW

**Review by nurul-umbhiya — APPROVED** [review]

File: includes/Vendor/SetupWizard.php [hunk 1]
```diff
@@ -118,6 +118,9 @@ public function frontend_enqueue_scripts() {
         wp_enqueue_script( 'jquery-tiptip' );
         wp_enqueue_script( 'jquery-blockui' );
         wp_enqueue_script( 'wc-enhanced-select' );
+
+        // Load map scripts.
+        dokan()->scripts->load_gmap_script();
     }
```
Comment (Aunshon) [inline includes/Vendor/SetupWizard.php:124]: Please check if  map box loads properly.

**Review by Aunshon — CHANGES_REQUESTED** [review]

File: includes/Vendor/SetupWizard.php (hunk 1 above)
Comment (devAsadNur) [reply includes/Vendor/SetupWizard.php:124]: > Please check if map box loads properly.

Bhai, both `Map Box` and `Google Map` are loading properly.

**Comment by shashwatahalder01** [comment]
Scenario:
map search suggestion not loading on setup wizard

Screenshot:
Screenshot at Jun 21 13-10-03

**Comment by devAsadNur** [comment]
> Scenario: map search suggestion not loading on setup wizard. console error exists.
> 
> Screenshot: Screenshot at Jun 21 13-10-03 Screenshot at Jun 21 13-13-03

The issue has been fixed by [this commit](https://github.com/weDevsOfficial/dokan/pull/1924/commits/f61a440a9bcdb579e0a6db088eaf7687d47c6db9).

**Comment by shashwatahalder01** [comment]
Scenario:
Showing redundant message or selected address outside seller  setup wizard modal

Screenshot:
Screenshot at Jun 22 08-51-30
Screenshot at Jun 22 08-51-52

**Review by Aunshon — APPROVED** [review]

**Pull request merged**